Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -903,3 +916,77 @@
 #if !defined(ENABLE_PREWARM_WEBPROCESS_ON_PROVISIONAL_LOAD)
 #define ENABLE_PREWARM_WEBPROCESS_ON_PROVISIONAL_LOAD 1
 #endif
+
+#if (PLATFORM(IOS) || PLATFORM(MAC) || PLATFORM(VISION)) && USE(APPLE_INTERNAL_SDK)
+#define HAVE_PASSKIT_INSTALLMENTS_IN_SDK 1
+#endif
+
+#if !defined(ENABLE_NETWORK_ISSUE_REPORTING) \
+    && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000) \
+    || ((PLATFORM(IOS) || PLATFORM(MACCATALYST)) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000) \
```

Comment:
Soooo close to just being IOS_FAMILY

Reply:
Yeah, the pesky builders have to be kept happy. We should do a revision once the bots have moved to fall release.